source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs

```diff
@@ -42,7 +42,7 @@ public CoordinateConversionViewModel()
         public static CoordinateConversionLibraryConfig AddInConfig = new CoordinateConversionLibraryConfig();
 
         // InputCoordinate
-        private string inputCoordinate = "70.49N40.32W";
+        private string inputCoordinate = Properties.Resources.StringInputCoord;
```

Comment:
this is still not back to the original line

private string inputCoordinate = "70.49N40.32W";